### PR TITLE
Feature/correct references of national statistics

### PR DESCRIPTION
**Pull request by dionisislolas**
### What

Correct references to “Accredited official statistic” to be “Accredited official statistics”
Change "NS status" to "AOS status"

### How to review

Check changes ok.

### Who can review

Anyone

### CODE REVIEW

File: src/audit-allowlist.json [hunk 1]
```diff
@@ -19,6 +19,14 @@
...
             "id": "CVE-2023-45133",
             "reason": "indirect dependency of a few babel packages. should be looked at as a seperate ticket. see: https://trello.com/c/alXeUZpI/1918-fix-dependency-issues-with-babel-traverse"
+        },
+        {
+            "id": "CVE-2024-4067",
+            "reason": "Micromatch, dependency of jest. should be looked at as a seperate ticket. context: https://github.com/micromatch/micromatch/issues/243"
+        },
+        {
+            "id": "CVE-2024-4068",
+            "reason": "Braces, dependency of micromatch, indirect dependency of jest. context: https://github.com/micromatch/micromatch/issues/243"
```
Comment (lindenmckenzie) [inline src/audit-allowlist.json:29]: Have you created tickets for this?

Comment (dionisislolas) [reply src/audit-allowlist.json:29]: No m, will do that now. Thank you Linden.

**Review by lindenmckenzie — APPROVED** [review]

File: src/audit-allowlist.json (hunk 1 above)
Comment (mr-nick17) [reply src/audit-allowlist.json:29]: Braces is resolved with v3.0.3

**Review by mr-nick17 — CHANGES_REQUESTED** [review]
Braces and micromatch issues have been resolved AKA passing audit with these versions 
```
    "braces": "^3.0.3",
    "micromatch": "^4.0.6"
```

**Review by mr-nick17 — APPROVED** [review]

**Pull request merged**